pkg/clients/clients.go

```diff
@@ -168,6 +169,10 @@ func SetScheme(crScheme *runtime.Scheme) error {
 		return err
 	}
 
+	if err := imagev1.AddToScheme(crScheme); err != nil {
```

Comment:
since you're adding the scheme in the Pull function you should be fine without adding it here. You may need to add the scheme in newBuilder too though

Reply:
I remove it from the Pull function, it doesn't work vice versa

Reply:
@elenagerman what was the issue having it in the Pull function? we shouldn't need to put it in the clients package anymore

Reply:
in general it just fails to create the object: "no kind is registered for the type v1.ImageStream in scheme \"pkg/clients/clients.go:312\"

Reply:
As @klaskosk mentioned, this should be done in the Pull function. Please reference this example - https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/hive/clusterdeployment.go#L232

Reply:
@elenagerman You should be able to do this in the pull function similar to the way it was done here - https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/clusterversion/clusterversion.go#L45. Both of these structs are coming from openshift/api so I imagine they should behave the say way

Reply:
all done: scheme was removed from the client. thanks a lot for advice and provided example!!!